default/package.json

```diff
@@ -15,11 +15,14 @@
     "eslint-plugin-sonarjs": "^0.6.0",
     "husky": "4.3.0",
     "lint-staged": "^10.5.4",
+    "material-ui-popup-state": "^1.8.2",
     "prettier": "^2.2.1",
     "react-app-rewire-yaml": "^1.1.0",
     "react-app-rewired": "^2.1.8"
   },
   "dependencies": {
+    "@algo-builder/algob": "^0.5.5",
```

Comment:
Why do we add here as a dependency? We added algob in `scripts`

Reply:
When I imported config from `algob.config.js`, it was required.

Reply:
No but you can link @algo-builder/algob here, instead of adding as a dependency (same as we do in /default). OR remove algob from scripts in both

Reply:
Sure.